<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>620</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="label_menu">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 102); 
font: 87 28pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>MENU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_show_all_payments">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>320</width>
     <height>50</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt;color:rgb(54,54,54);
font: 75 14pt &quot;Arial&quot;;
background-color: rgb(255,0,102);
font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>SHOW ALL PAYMENTS</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_warning">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255); font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_show_daily_payment">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>320</width>
     <height>50</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(54,54,54);
font: 75 14pt &quot;Arial&quot;;
background-color: rgb(255,0,102);
font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>SHOW DAILY PAYMENT</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_add_payment">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>320</width>
     <height>50</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt;color:rgb(54,54,54);
font: 75 14pt &quot;Arial&quot;;
background-color: rgb(255,0,102);
font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>ADD PAYMENT</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_delete_payment">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>320</width>
     <height>50</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt;color:rgb(54,54,54);
font: 75 14pt &quot;Arial&quot;;
background-color: rgb(255,0,102);
font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>DELETE PAYMENT</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_others">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>320</width>
     <height>50</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt;color:rgb(54,54,54);
background-color: rgb(255,0,102);
font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>OTHERS</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_quit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>510</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Arial Black&quot;;
color:rgb(255,0,102);</string>
   </property>
   <property name="text">
    <string>QUIT</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
